airbyte-integrations/connectors/source-clickup-api/source_clickup_api/manifest.yaml

```diff
@@ -8,250 +8,1064 @@ check:
+    list_custom_fields: true
+    team_custom_fields: true
+    space_tags: true
+    team_goals: true
+    list_comments: true
+  yamlComponents:
+    global:
+      - authenticator
+  testedStreams:
+    user:
+      streamHash: 7176ff53062a223428ecb44db1db66e578621c5e
+      hasResponse: true
+      responsesAreSuccessful: true
+      hasRecords: true
+      primaryKeysArePresent: true
+      primaryKeysAreUnique: true
+    team:
+      streamHash: af69592494618f0fe3eae8065fab8a9f541264d4
+      hasResponse: true
+      responsesAreSuccessful: true
+      hasRecords: true
+      primaryKeysArePresent: true
+      primaryKeysAreUnique: true
+    list:
+      streamHash: e7a394b78d97097d82d3b6205f00a7776efcb249
+      hasResponse: true
+      responsesAreSuccessful: true
+      hasRecords: true
+      primaryKeysArePresent: true
+      primaryKeysAreUnique: true
+    space:
+      streamHash: 460deeb83e207d5cd0fb55895563c1731c93cd0d
+      hasResponse: true
+      responsesAreSuccessful: true
+      hasRecords: true
+      primaryKeysArePresent: true
+      primaryKeysAreUnique: true
+    folder:
+      streamHash: b6741eb32c353db9d9d4965df443c789e34441f6
+      hasResponse: true
+      responsesAreSuccessful: true
+      hasRecords: true
+      primaryKeysArePresent: true
+      primaryKeysAreUnique: true
+    task:
+      streamHash: 62ed778c46b789e6b6301ed5b7a1a628a2f723d6
+      hasResponse: true
+      responsesAreSuccessful: true
+      hasRecords: true
+      primaryKeysArePresent: true
+      primaryKeysAreUnique: true
+    time_tracking_tags:
+      streamHash: b9fc3e6eedb31e0ad3345412ff47048041f0e4e1
+      hasResponse: true
+      responsesAreSuccessful: true
+      hasRecords: true
+      primaryKeysArePresent: true
+      primaryKeysAreUnique: true
+    time_tracking:
+      streamHash: 5c0dacb941b6fd23301adad0ce79b2c469cf3193
+      hasResponse: true
+      responsesAreSuccessful: true
+      hasRecords: true
+      primaryKeysArePresent: true
+      primaryKeysAreUnique: true
+    list_custom_fields:
+      streamHash: 8541a6102f84cb9baf8f07954859dc5377f51833
+      hasResponse: true
+      responsesAreSuccessful: true
+      hasRecords: true
+      primaryKeysArePresent: true
+      primaryKeysAreUnique: false
+    team_custom_fields:
+      streamHash: dae304f3af38127a8b6af8852679ce7c82483888
+      hasResponse: true
+      responsesAreSuccessful: true
+      hasRecords: true
+      primaryKeysArePresent: true
+      primaryKeysAreUnique: true
+    space_tags:
+      streamHash: 1a66b563a72003ff4c5bf48e7fecb187b2dd0d6f
+      hasResponse: true
+      responsesAreSuccessful: true
+      hasRecords: true
+      primaryKeysArePresent: true
+      primaryKeysAreUnique: true
+    team_goals:
+      streamHash: 921a14d155ca527219909c94a0db0bfdde3d0b14
+      hasResponse: true
+      responsesAreSuccessful: true
+      hasRecords: true
+      primaryKeysArePresent: true
+      primaryKeysAreUnique: true
+    list_comments:
```

Comment:
These blocks are populated when you do test reads in Builder, and we want to see them in most contributions. Good work @btkcodedev

Reply:
/cc @lmossman it works!